setup.cfg

```diff
@@ -17,7 +17,7 @@ install_requires =
     angr[angrDB] == 9.2.26.dev0
     qtconsole
     ipython
-    PySide6 < 6.4.0
+    PySide6 >= 6.4.0.1
```

Comment:
Maybe we want this to be `~= 6.4.0` instead of assuming 6.5 will just work?

Reply:
I don't expect even 6.4.1 to work tbh

Reply:
`~= 6.4.0.1` would be 6.4.0.x but not `6.4.1`